src/data/Data.js

```diff
@@ -769,22 +768,22 @@ export const WorkspaceData = _.flow(
 
     const loadWdsData = useCallback(async () => {
       // Try to load the proxy URL
-      if (!wdsProxyUrl || wdsProxyUrl.status !== 'Ready') {
+      if (!wdsProxyUrl || !['Ready', 'Error'].includes(wdsProxyUrl.status)) {
```

Comment:
This is getting a bit grosser before it gets better.  I hope to still tighten this logic up a bit, but I thought it prudent not to tie up a fix to the polling behavior behind the cleanup.